README.md

```diff
@@ -1,12 +1,3 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
-
-To get started:
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/google/software-product-sprint.git`
-- Open the GitHub setup walkthrough: `teachme ~/software-product-sprint/walkthroughs/week-0-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+This repo contains Kiba's portfolio and SPS projects."
```

Comment:
nit: the double quote at the end of the sentence looks extra and should probably be removed. (I use "nit:" to prefix nit picky comments that are purely cosmetic)